tasks/css.js

```diff
@@ -36,6 +37,8 @@ module.exports = function (config, base, outputName, input) {
 
   Object.keys(config.breakpoints).forEach(breakpoint => {
     cssVars[`breakpoint-${breakpoint}`] = config.breakpoints[breakpoint]['min-width'];
+    const minmax = (config.breakpoints[breakpoint].smallest) ? 'max' : 'min';
+    customMedia[breakpoint] = `(${minmax}-width: ${config.breakpoints[breakpoint]['min-width']})`;
```

Comment:
With `max-width` you'd probably want `config.breakpoints[breakpoint]['min-width']` to change. In the new grid I have that bit. Also changed `smallest` to `default` since it wouldn't make since to have desktop be smallest.